style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/creating-your-bot/adding-more-commands.md

});
```

> Before doing anything else, make a property to store the prefix you've configured. Instead of `const config = ...`, you can use destructuring to extract the prefix variable from the config file and the token as well while you're at it.

Reviewer: ```suggestion
Before doing anything else, make a property to store the prefix you've configured. Instead of `const config = ...`, you can use destructure the config file to extract the prefix and token variables.
```

Now the ping command will trigger whenever the message _starts with_ `!ping`! Sometimes this is what you want, but other times, you may want to match only exactly `!ping` - it varies from case to case, so be mindful of what you need when creating commands.

::: warning

> Be aware that this will also match `!pingpong`, `!pinguin`, and the like. This is not a huge problem, for now, so don't worry; you'll see better ways to check for commands later.

Reviewer: ```suggestion
Be aware that this will also match `!pingpong`, `!pinguin`, and the like. This may or may not be a problem for you, but don't worry; you'll see better ways to check for commands later.
```

	</discord-message>
</div>

> You can, of course, modify this to your liking. You may want to also display the date the server was created or the server's region. You would do those in the same manner - use `message.guild.createdAt` or `message.guild.region`, respectively.

Reviewer: ```suggestion
Of course, you can modify this to your liking. You may want to also display the date the server was created or the server's region. You would do those in the same manner - use `message.guild.createdAt` or `message.guild.region`, respectively.
```
